### ISSUE TITLE
Editor under "body" has distorted view

### ISSUE THREAD

**Issue by ash007-postman** [a85/POSTMan-Chrome-Extension#6760]
**Describe the bug**
After sending the request, the lines are overlapping in "Body" editor

**To Reproduce**
1. click on  'Body"
2. Lines are overlapping

**Expected behavior**
Lines shouldn't be overlapping

**App information (please complete the following information):**
 - Postman Version [e.g. 7.2.0]

![Screenshot 2019-06-20 at 12 17 29 PM](https://user-images.githubusercontent.com/41741228/59827287-eb423a00-9355-11e9-9524-00b2ea14290e.png)

**Comment by kamalaknn** [comment]
Hi @ash007-postman 

We are not able to reproduce this issue from our end.

Can you get more information from the user to help us identify the issue?

1. Is this reproduceable consistently?
2. Is this new on Postman 7.2.2 or has the same text and request worked in previous versions?
3. Can you send a screen recording of the issue or a collection with saved request containing request body that can help us reproduce this?

**Comment by daehee75** [comment]
Hi @kamalaknn 

This is Dean who created this issue.

This situation is reproducible consistently and it can be found in previous versions.
 
But I'm not sure only Postman app's problem. Because I can find same situation in some online editor like attached image.
![image](https://user-images.githubusercontent.com/21234550/59830072-d1b2e980-9379-11e9-937e-1d340fa5fcff.png)
I think it works on some app with a specific editor engine.
Can you help us?
Thank you.

Regards,
Dean

**Comment by kamalaknn** [comment]
Hey @daehee75 

Like you have mentioned I don't think this is a bug in Postman as such since you are also seeing it on a different tool on browser. 

It looks like there is some issue with how these editors are rendering in your environment. Can you share more information on your system hardware, operating system and browser so we can check if there are any known issues related to it?

**Comment by daehee75** [comment]
Hi @kamalaknn 

Device : Macbook Pro (15-inch, 2017)
OS : macOS Mojave (10.14.5)
Browser : Chrome 74.0.3729.169 (64bit)

in Safari, there is no bug.

**Comment by harryi3t** [comment]
@daehee75 
Could you give a try to our canary build which you can get from https://www.getpostman.com/downloads/canary
It has an updated version of the underlying platform (Electron) on which Postman is built.

**Comment by daehee75** [comment]
@kamalaknn 

I tried PostmanCanary but still remain this bug unfortunately. :(

![image](https://user-images.githubusercontent.com/21234550/59905593-98de4780-9441-11e9-873d-a7947cbdee68.png)

**Comment by eugen1j** [comment]
@daehee75 I have the same problem. Did you manage to solve it?

**Comment by hkdollar** [comment]
> @daehee75 I have the same problem. Did you manage to solve it?

No. It's still open issue. :(

**Comment by daehee75** [comment]
> @daehee75 I have the same problem. Did you manage to solve it?

No. It's still open issue. :(

**Comment by daehee75** [comment]
> @daehee75 I have the same problem. Did you manage to solve it?

 No. It's still open issue.
![image](https://user-images.githubusercontent.com/21234550/82286142-f4111500-99d7-11ea-9996-c606fb9f721c.png)

**Comment by eugen1j** [comment]
@daehee75 I manage to solve the problem here https://github.com/ajaxorg/ace/issues/4300

TL;DR: I had broken font `Monaco` on my computer.

**Comment by daehee75** [comment]
I fixed font Monaco and solved the problem on your information.

Thank you, @eugen1j.   👍 


> @daehee75 I manage to solve the problem here [ajaxorg/ace#4300](https://github.com/ajaxorg/ace/issues/4300)
> 
> TL;DR: I had broken font `Monaco` on my computer.

**Comment by arlemi** [comment]
Thanks for investigating and letting us know @eugen1j ! Closing this issue now. 

FYI we'll soon move that editor to use `Monaco editor` instead of `Ace editor`.

**Comment by daehee75** [comment]
> Thanks for investigating and letting us know @eugen1j ! Closing this issue now.
> 
> FYI we'll soon move that editor to use `Monaco editor` instead of `Ace editor`.

Thank for your information @arlemi
